generator/postgres/query_set.go

```diff
@@ -57,6 +57,7 @@ func getColumnsMetaData(db *sql.DB, schemaName string, tableName string) ([]meta
 	query := `
 select  
     attr.attname as "column.Name",
+	dsc.description as "column.Comment",
```

Comment:
Since we are already using `obj_description` to get table column, maybe we should use `col_description` to get column description.

Reply:
Also note the ident, tabs vs spaces.